mainapp/templates/mainapp/request_form.html

```diff
@@ -131,7 +131,7 @@ <h3 id="myModalLabel">Allow Location - നിങ്ങളുടെ സ്ഥല
 	})
 
   </script>
-  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDpPl_JNYYBqaWVWDXCJ3rCJ9H92kQsLVs&libraries=places&callback=initMap"
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDdPKYd4vwgQY-OmF0Vn22wJu0EcDyi1MQ&libraries=places&callback=initMap"
```

Comment:
it is good to keep in env files. This will surely get caught by some crawl bots and mis-use!

Reply:
this is served to the client in the HTML anyways, so it wouldn't make a difference to crawl-bots. This is just how map keys work.

However, for code organization, etc. it would be nice to put it in `settings` at least, and make it over-ridable with an environment variable. However, that probably does not need to hold up a merge if this is urgent.